setup.py

```diff
@@ -15,12 +15,21 @@
     include_package_data=True,
     py_modules=['trie'],
     setup_requires=['setuptools-markdown'],
+    python_requires='>=2.7, !=3.5.2',
```

Comment:
Lower bound should be `>=3.5` and an upper bound of  `<4` should also be included.

Reply:
Should be `>=3.5.3` to avoid with some early py3.5 bugs